src/main/cpp/include/query_operations/variant_operations.h

```diff
@@ -88,6 +88,76 @@ class RemappedVariant : public RemappedDataWrapperBase
     unsigned m_queried_field_idx;
 };
 
+typedef std::tuple<const std::vector<int>*, uint64_t, bool> GenotypeForMinValueResultTuple;
+
+template<class DataType>
```

Comment:
Can you add some comments on the class? Just to understand the responsibility of this class.

Reply:
Done